### PR TITLE
ISPN-14529 query docs: clarify attributes for sorting queries

### PR DESCRIPTION
**Pull request by dvagnero**
https://issues.redhat.com/browse/ISPN-14529

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ There are two annotations that control the entities and fields that {brandname}
 
 |`store`
 |Allows {brandname} to store fields in indexes so you can use them for projections.
-|`Store.YES` or `Store.NO`
+|`Store.YES` or `Store.NO`. Use `Store.YES` and set `sortable = true` for fields that need to be used for sorting.
```
Comment (fax4ever) [inline documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc:22]: The store options does not exist anymore.
The new way to mark a field sortable is to set `sortable = true`

File: documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc [hunk 2]
```diff
@@ -43,6 +43,8 @@ For example, the following schema uses the `@Field` annotation:
 ----
 include::protobuf/indexed_field.protobuf[]
 ----
++
+By including `store = Store.YES` and `sortable = true` in the `@Field` annotation, you can use the `street` field for sorting queries without encountering warning messages or unexpected results.
```
Comment (fax4ever) [inline documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc:47]: as above

File: documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc (hunk 1 above)
Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/con_indexing_annotations.adoc:22]: Hi @fax4ever thank you for checking the PR. I guess you're speaking about Infinispan 15? This is a clarification for Infinispan 13 and previous versions so the old way.

**Comment by dvagnero** [comment]
@wfink could you please take a look?

**Comment by wfink** [comment]
LGTM

**Review by wfink — APPROVED** [review]

**Pull request merged**